.github/workflows/publish_github.yml

```diff
@@ -1,15 +1,12 @@
-name: Node.js Github Package
+name: Publish
 
 on:
-  pull_request:
-    branches:
-      - main
-  push:
+  pull_request_target:
```

Comment:
we need to use pull_request_target due to this blog post https://github.blog/changelog/2021-02-19-github-actions-workflows-triggered-by-dependabot-prs-will-run-with-read-only-permissions/